libs/gl-client/src/lsps/lsps0/common_schemas.rs

```diff
@@ -144,6 +144,94 @@ impl<'de> Deserialize<'de> for MsatAmount {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct ShortChannelId {
```

Comment:
We already have a pretty much identical type in cln-rpc: https://github.com/ElementsProject/lightning/blob/77e3c3a38e808986d4515d92302ed7666e8bc99a/cln-rpc/src/primitives.rs#L130-L183

Would it be possible not to duplicate that type? We also have an amount type, but I'm not sure it's stringified version matches what you need for the LSPS, so if that doesn't work out, that's ok. SCIDs on the other hand should be pretty much identical.

Reply:
The class is equivalent and replacing it would be trivial. 
The gl-client is not depending on `cln-rpc` yet. 

If you confirm you want to include it as a dependency I'll make the change

Reply:
Yeah, you're right, adding a dependency on `cln-rpc` while clean, also means we pull in stuff like the UnixStream dependency which we definitely don't want in the client.

Reply:
I've copied the ShortChannelId from cln-rpc.

If we ever split out the primitives we should be able to use them without creating a breaking change